Extractors.md

```diff
@@ -0,0 +1,72 @@
+
+
+# Extractors
+
+Extractors are a kind of jobs that allows to extract the required columns from the sources. 
```

Comment:
I think you need to add a sentence after this one, I would suggest the following: 

_and maps them to the Event it is extracting.

 An Extractor is defined by Event type it is extracting and the Source where the information needed to build the extract Event is._